src/main/java/domain/LottoManager.java

```diff
@@ -0,0 +1,19 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import util.RandomNumbersGenerator;
+
+public class LottoManager {
+    public LottoWallet generateLottos(final int lottoCount) {
+        List<Lotto> lottos = new ArrayList<>();
+        for (int i = 0; i < lottoCount; ++i) {
+            lottos.add(new Lotto(RandomNumbersGenerator.generateUniqueNumbers(LottoNumber.MIN, LottoNumber.MAX, 6)));
+        }
+        return new LottoWallet(lottos);
+    }
+
+    public float calculateRevenue(WinningResult winningResult, Money money) {
+        return (float) winningResult.getTotalPrices() / money.getValue();
```

Comment:
```suggestion
    public float calculateRevenue(WinningResult winningResult, Money money) {
        return winningResult.calculateRevenue(money);
```
WinningResult에 수익률이 얼만지 물어보는 관점의 설계는 어떨까요?

Reply:
getter로 값을 가져와 연산을 수행하는 것 보다 
해당 도메인에게 메시지를 던져 원하는 것을 얻는 방법이 좋겠네요.
좋은 제안 감사합니다.